purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -886,6 +886,22 @@ class Purchases internal constructor(
         )
     }
 
+    /**
+     * Subscriber attribute associated with the CleverTap ID for the user
+     * Required for the RevenueCat CleverTap integration
+     *
+     * @param cleverTapID null or an empty string will delete the subscriber attribute.
+     */
+    fun setCleverTapID(cleverTapID: String?) {
```

Comment:
We need to add this to the API tests.

Reply:
We probably want to add a test in `PurchasesTest` as well. I see we don't have any for these methods setting attributes, but it would be a good idea to add them.

Reply:
AHhh yeah. Thanks for the reminder!

Reply:
Added to API tester!